scripts/exregional_run_post.sh

```diff
@@ -296,7 +296,17 @@ zero exit code."
 # A separate ${subh_fhr} is needed for subhour post.
 #-----------------------------------------------------------------------
 #
+
 len_fhr=${#fhr}
+if [ ${len_fhr} -eq 9 ]; then
```

Comment:
Why is this hard coded? Would be nice to use a variable from config.sh

Reply:
When sub-hour forecast is turned on, the forecast file has 9-digit time stamp: hhh-mm-ss. It is not defined by workflow. This script just use it to figure out the time for post.

Reply:
Ok, it is the length of the "fhr" string. However, it is still easy to confuse it with the forecast length.  Would it be better to change it to something like "fhr_digits"?

Reply:
Thanks, it is helpful to add a comment there.